src/language/utils.ts

```diff
@@ -90,10 +90,16 @@ module Lint {
         });
     }
 
-    export function isBlockScopedVariable(node: ts.VariableDeclaration): boolean {
-        // determine if the appropriate bit in the parent (VariableDeclarationList) is set,
-        // which indicates this is a "let" or "const"
-        return (Math.floor(node.parent.flags / ts.NodeFlags.Let) % 2) === 1
-            || (Math.floor(node.parent.flags / ts.NodeFlags.Const) % 2) === 1;
+    /**
+     * Determines if the appropriate bit in the parent (VariableDeclarationList) is set,
+     * which indicates this is a "let" or "const".
+     */
+    export function isBlockScopedVariable(node: ts.VariableDeclaration | ts.VariableStatement): boolean {
+        const flags = (node.kind === ts.SyntaxKind.VariableDeclaration)
+            ? (<ts.VariableDeclaration> node).parent.flags
+            : (<ts.VariableStatement> node).declarationList.flags;
+
+        return (Math.floor(flags / ts.NodeFlags.Let) % 2) === 1
```

Comment:
to get this to typecheck & pass tslint, you have to do something like:

```ts
/* tslint:disable:no-bitwise */
return ((flags & ts.NodeFlags.Let) !== 0) || ((flags & ts.NodeFlags.Const) !== 0);
/* tslint:enable:no-bitwise */
```

is that preferable to the current implementation?